### PR TITLE
Fix incl command whitespace bug

### PR DESCRIPTION
**Pull request by Bill-1**
# Pull Request Template

## Issue
[For multiple issues see [here](https://docs.github.com/en/issues/tracking-your-work-with-issues/using-issues/linking-a-pull-request-to-an-issue)]

Fix https://github.com/AY2425S2-CS2103T-T14-4/tp/issues/124

## Type
- [ ] Feature
- [X] Bug fix
- [ ] Doc update
- [ ] Test update
- [ ] Refactor

## Checklist
- [ ] UG is updated
- [ ] DG is updated
- [ ] `AddressBookParserTest` is updated
- [ ] `*CommandTest`is updated
- [ ] `*CommandParserTest` is updated
- [ ] I followed Git conventions
- [ ] I added milestone to issue and PR
- [ ] There is no `checkstyle` warnings
- [ ] I tested the patch locally
- [ ] The spacing for all the messages are OK

## Testing
- [ ] Unit
- [ ] Integration (e.g. `ModelManagerTest`)
- [ ] System (compile and run the app)

## Description
- I did ...
- I tested by ...
- I documented ...

## Plagiarism
- [ ] Own work
- [ ] AI generated; I changed and tested the code
- [ ] Other's work; I quoted authors in code

## Git conventions
[See [here](https://se-education.org/guides/conventions/git.html#commit-message-body)]
- Every commit must have a well-written commit message subject line.
- Use the imperative mood in the subject line.
- Capitalize the first letter of the subject line.
- Do not end the subject line with a period.
- Commit messages for non-trivial commits should have a body giving details of the commit.

## Code quality
- [ ] Logging
- [ ] Exceptions
- [ ] Assertions
- [ ] Booleans prefixed with `is` or `has`

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/IncludePersonCommandParser.java [hunk 1]
```diff
@@ -22,24 +24,32 @@ public class IncludePersonCommandParser implements Parser<IncludePersonCommand>
...
         return new IncludePersonCommand(namePredicate);
     }
 
+    /**
+     * Processes keywords from the ArgumentMultimap. Each keyword value is split by
+     * whitespace.
+     *
+     * @param value List of keyword values
+     */
+    private List<String> processKeywords(Optional<String> value) {
```
Comment (xumarcus) [inline src/main/java/seedu/address/logic/parser/IncludePersonCommandParser.java:52]: Refactor this to commons, check edge cases like `FindCommand`

**Review by xumarcus — CHANGES_REQUESTED** [review]
Add test to parser for this case

File: docs/UserGuide.md [hunk 2]
```diff
@@ -208,11 +208,14 @@ Examples:
 
 ### Including your contacts by name to specific group: `incl`
 
-Adding people with matching name from your contacts to the list.
+Adding contacts with matching name from your contacts to the list.
 
-Format: `incl n/NAME_KEYWORD`
+Format: `incl {n/NAME_KEYWORD [NAME_KEYWORD]...}`
```
Comment (BladerX11) [inline docs/UserGuide.md:213]: Don't need `{}` as it is required and not an option between 2

File: docs/UserGuide.md [hunk 3]
```diff
@@ -208,11 +208,14 @@ Examples:
...
 
 * `NAME_KEYWORD` is same as defined above.
+* Supply at least one of `n/NAME_KEYWORD`
+* It includes persons whose name contain at least one keyword. e.g. Providing `n/Hans Bo` includes `Hans Gruber`, `Bo Yang`
```
Comment (BladerX11) [inline docs/UserGuide.md:217]:
```suggestion
* It includes contacts whose name contain at least one keyword. e.g. Providing `n/Hans Bo` includes `Hans Gruber`, `Bo Yang`
```

File: docs/UserGuide.md
Comment (BladerX11) [inline docs/UserGuide.md:1]: Maybe add an example of first performing a find that does not have the person you want to `incl` screenshot of expected output after. Can look existing command examples in the master branch

File: src/main/java/seedu/address/logic/parser/IncludePersonCommandParser.java (hunk 1 above)
Comment (BladerX11) [reply src/main/java/seedu/address/logic/parser/IncludePersonCommandParser.java:52]: can probably take the `FindCommand` one and put it in `StringUtil`

**Review by BladerX11 — CHANGES_REQUESTED** [review]
